src/ia-anniversary-banner.ts

```diff
@@ -229,41 +106,16 @@ export class IaAnniversaryBanner extends LitElement {
         position: relative;
       }
 
-      .base,
-      .text {
-        position: absolute;
-        right: 0;
-        left: 0;
-        top: 0;
-        bottom: 0;
-      }
-
-      .base svg,
-      .text svg {
-        height: 100%;
-        width: 100%;
-      }
-
-      .mobile-banner {
-        display: none;
+      @media screen and (max-width: 767px) {
+        .wayforward.banner {
+          background-image: url('https://archive.org/download/ia-25-wf/wf-banner-mobile.png');
```

Comment:
QUESTION: Did we want to lock this to the production instance of the banner? e.g. Won't it prevent testing of staging instances?

Reply:
no staging instance, this is just image asset hosted on the site.